sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -359,6 +397,9 @@ def _store_extra_properties_in_sbol3(self, doc: sbol3.Document, comp: Component_
         :param record: GenBank SeqRecord instance for the record which contains extra properties
         """
         comp.genbank_record_id = record.id
+        # set dblinks from the dbxrefs property of biopython
+        if record.dbxrefs:
+            comp.genbank_dblink = "::".join(record.dbxrefs)
```

Comment:
I don't understand what this is, and the comment above isn't helping me. Can you please expand the comment to explain the purpose, maybe link to a test example as well?
I'm pretty sure this is about structured comments, but there's nothing in the code here that's explaining that.

Reply:
My bad, I forgot to explain this; I have added comments in the latest commit; this is just about how biopython stores dbxrefs in a list, and how '::' is used as a delimiter to store the list as a flat string and then reset back to a list during round trip conversion.